app-editors/vis/vis-0.3.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+DESCRIPTION="Vis aims to be a modern, legacy free, simple yet efficient vim-like editor."
+HOMEPAGE="https://github.com/martanne/vis"
+SRC_URI="https://github.com/martanne/vis/archive/v${PV}.tar.gz -> vis-0.3.tar.gz"
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="-selinux tre +curses"
+
+#TODO: virtual/curses; >=dev-lang/lua-5.2 (needed for syntax highlighting and settings)
+DEPEND="dev-libs/libtermkey tre? ( dev-libs/tre:* ) curses? ( sys-libs/ncurses:= )"
```

Comment:
1. One dep per line to improve readability.
2. Why are you putting `:*` on tre? Quite uncommon for a regular program using a C library.
3. `:0=` for ncurses, unless it can work without headers.